### DISCUSSION TITLE
in book page, list all my quotes and comments for unregistered users

### DISCUSSION THREAD

**Discussion by fivestone** [bookwyrm-social/bookwyrm#3358] [Ideas]
In a self-host bookwyrm, although all my comment are public, it seems that unregistered users can only visit them via separated links, or browse them in my activity page mixed with the comments of all the books.

Is it possible, in the page of a certain book, unregistered users can browse all the quotes and comments of mine about only this book?

Thanks a lot.

**Comment by VMPYRC** [comment]
this sounds similar to #1275 

i agree, it'd be nice to share AND publicly view comments and quotes on a book's page.
